Update draft-secure-credential-transfer.md (#23)
dimmyvi/secure-credential-transfer: draft-secure-credential-transfer.md

Bad Request - invalid request has been passed (can not parse or required fields missing).

`401`

> Unauthorized - calling device is not authorized to create a mailbox. E.g. a device presented the incorrect deviceClaim or deviceAttestation provided is invalid.

Reviewer: ```suggestion
Unauthorized - calling device is not authorized to create a mailbox. E.g. a device presented an invalid deviceClaim or deviceAttestation.
```
What do you think about this?
